### DISCUSSION TITLE
unable to enforce https in github page, can anyone help?

### DISCUSSION THREAD

**Discussion by itwas-22** [community/community#37708] [Other Feature Feedback, Questions, & Ideas]
![image](https://user-images.githubusercontent.com/66280259/198828996-e06f27db-5a57-4717-8f30-787d27736d57.png)

**Comment by yoannchaudet** [comment]
Hello,

We cannot enforce HTTPS until a TLS certificate has been issued. Since you masked part of the domain I cannot go investigate further. The two subdomains you have setup on the `github.fi` are not properly setup today as they point to an IP address that is not GitHub Pages.

Please refer to https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-a-subdomain.
